cmd/jaeger/internal/integration/kafka_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+func TestKafkaStorage(t *testing.T) {
+    integration.SkipUnlessEnv(t, "kafka")
+
+    collectorConfig := "../../collector-with-kafka.yaml"
+    ingesterConfig := "../../ingester-remote-storage.yaml"
+
+    collector := &E2EStorageIntegration{
+        SkipStorageCleaner: true,
+        ConfigFile: collectorConfig,
+        StorageIntegration: integration.StorageIntegration{
+            GetDependenciesReturnsSource: true,
+            SkipArchiveTest: true,
+        },
+    }
+
+    // Initialize and start the collector
+    collector.e2eInitialize(t, "kafka-collector")
+
+
+    ingester := &E2EStorageIntegration{
+        ConfigFile: ingesterConfig,
+        StorageIntegration: integration.StorageIntegration{
+            CleanUp: purge,
+            GetDependenciesReturnsSource: true,
+            SkipArchiveTest: true,
+        },
+    }
+
+	// Initialize and start the ingester
+    ingester.e2eInitialize(t, "kafka-ingester")
```

Comment:
What is the meaning of string argument? Isn't it supposed to be storage name in the config?